spec/controllers/callbacks_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe CallbacksController, type: :controller do
+
+end
```

Comment:
teaching moment! what's this one for? 

current test is pending, should we add test?

@ilyakrasnov